[IMP] subscriptions: updated reports doc for 17 (#7365)
odoo/documentation: content/applications/sales/subscriptions/reports.rst

in the database.

.. note::
   The :guilabel:`Default filter` and :guilabel:`Shared` options are **not** required. Only *one* of
   those options can be selected at a time.

To save that filter(s), click :guilabel:`Save` in the :guilabel:`Favorites` section of the drop-down
filter menu.

When clicked, that saved filter appears beneath the :guilabel:`Favorites` column of the drop-down
filter menu, and a :guilabel:`⭐ (gold star)` icon appears beside the favorite filter's name in the
search bar.

Views
-----

On the :guilabel:`Subscription Analysis`, :guilabel:`MRR Breakdown`, and :guilabel:`MRR Analysis`
reporting pages, three different view options are located in the upper-right corner.

.. note::
   There are *no* other view options available on the :guilabel:`Retention Analysis` reporting page.

The available view options, from left to right, are:

.. image:: reports/subscriptions-analysis-page-view-options.png
  :align: center
  :alt: The different view options available on the Subscriptions Analysis page.

- :guilabel:`Graph`
- :guilabel:`Pivot`
- :guilabel:`List`

Each view has its own series of related view-specific visual options.

Graph view
~~~~~~~~~~

> With the graph view selected the following options appear between the search bar and visual

Reviewer: ```suggestion
With the graph view selected, the following options appear between the search bar and visual
```

   Spreadsheet` button are *not* available.

Measures
--------

Each reporting page has its own metric-specific :guilabel:`Measures` drop-down menu of data-related
options to choose from, located in the upper-left corner, above the visual representation of
metrics.

.. image:: reports/subscriptions-measures-drop-down.png
  :align: center
  :alt: The standard measures drop-down menu in the Odoo Subscriptions application.

When clicked, a series of selectable measures become available, via a drop-down menu. When any of
the options are selected from a :guilabel:`Measures` drop-down menu, the chosen metrics related
to that specific measure appear on the reporting page.

.. note::
  For more information on the different measures that can be utilized on each reporting page, refer
  to the :ref:`specific reporting page breakdowns <subscriptions/reports/reporting>` found below in
  this documentation.

Insert in spreadsheet
---------------------

Beside the :guilabel:`Measures` drop-down menu, there is an :guilabel:`Insert in Spreadsheet`
button.

When clicked, the ability to add the configured data currently being showcased on the reporting page

> into a new or pre-existing spreadsheet (or dashboard) becomes available, via a pop-up window.

Reviewer: ```suggestion
into a new or pre-existing spreadsheet or dashboard becomes available, via a pop-up window.
```

clicked, a series of metric-related options become available.

.. image:: reports/subscriptions-retention-analysis-measures.png
  :align: center
  :alt: The measures drop-down menu of the Retention Analysis page.

The metric-related options in the :guilabel:`Measures` drop-down menu on the :guilabel:`Retention
Analysis` reporting page are:

- :guilabel:`Amount to invoice`
- :guilabel:`Margin`
- :guilabel:`Margin (%)`
- :guilabel:`Prepayment percentage`
- :guilabel:`Shipping Weight`
- :guilabel:`Unpaid Amount`
- :guilabel:`Count`

.. note::
  The :guilabel:`Count` measure option is selected by default.

To the right of the :guilabel:`Measures` drop-down menu on the :guilabel:`Retention Analysis` page
is an additional drop-down menu containing different time periods. The default time period is
:guilabel:`Month`.

When clicked, a drop-down menu of various time period options become available.

.. image:: reports/subscriptions-retention-analysis-time-periods.png
  :align: center
  :alt: The time period drop-down menu of the Retention Analysis page.

The time period options are:

- :guilabel:`Day`
- :guilabel:`Week`
- :guilabel:`Month`
- :guilabel:`Year`

> When a time period option from this drop-down menu is selected, the data in the :guilabel:`Retention
> Analysis` reporting page showcases data for the configured measures and filters within that time
> period.

Reviewer: ```suggestion
When a time period option from this drop-down menu is selected, the :guilabel:`Retention
Analysis` reporting page showcases data for the configured measures and filters within that time
period.
```

MRR breakdown
-------------

To access the :guilabel:`MRR Breakdown` reporting page, navigate to :menuselection:`Subscriptions
app --> Reporting --> MRR Breakdown`.

By default, the data displayed on the :guilabel:`MRR Breakdown` reporting page is in graph view,
with the :guilabel:`Bar Chart` option and the :guilabel:`Stacked` option selected.

There is also a default filter in the search bar for :guilabel:`Event Date: Month > Event Type`.

.. image:: reports/subscriptions-mrr-breakdown-default.png
  :align: center
  :alt: The default appearance of the MRR Breakdown reporting page in Odoo Subscriptions.

When the :guilabel:`Measures` drop-down menu on the :guilabel:`MRR Breakdown` reporting page is
clicked, a series of metric-related options become available.

.. image:: reports/subscriptions-mrr-breakdown-measures.png
  :align: center
  :alt: The default appearance of the MRR Breakdown reporting page in Odoo Subscriptions.

The metric-related options in the :guilabel:`Measures` drop-down menu on the :guilabel:`MRR
Breakdown` reporting page are:

- :guilabel:`Active Subscriptions Change`
- :guilabel:`ARR Change`
- :guilabel:`MRR Change`
- :guilabel:`Count`

.. note::

>    The :guilabel:`MRR Change` measure option is selected by default.

Reviewer: Can we change this if we want, so another one is the default?

Author: Yes. If you select a different 'Measure' option, and then click save that search under the "favorites" menu, and click the 'Default filter,' it will be the new default. I'll add that in this note admonition now. Thanks for the suggestion!

====================
Subscription reports
====================

Understanding how well subscriptions are performing is of the utmost importance.

The Odoo *Subscriptions* app provides a collection of detailed, fully-customizable reports
showcasing various subscription metrics and analytics. This knowledge can be used to improve

> decision-making and enhance subscription-related strategies.

Reviewer 2: Please be more specific and remove sales-sounding language.

Author: Completely rewrote the intro to remove any sales-sounding language, and provided specific mentions of each reporting page, along with how it can be used.

The Odoo *Subscriptions* app provides a collection of detailed, fully-customizable reports
showcasing various subscription metrics and analytics. This knowledge can be used to improve
decision-making and enhance subscription-related strategies.

Reporting page elements
=======================

All subscription-based reporting pages can be accessed via the :guilabel:`Reporting` header
menu in the *Subscriptions* application.

The following describes elements found on each reporting page.

Filters and Group By
--------------------

Filters are used to narrow down metrics to show specific analytics, whereas groupings (via the
:guilabel:`Group By` option) are used to gather specific sections' data into groups for more
organized analysis.

This section refers to both filters and groupings, as a combination of the two can be saved in the
:guilabel:`Favorites` section.

To modify the results being shown on any reporting page, click the :guilabel:`▼ (downward facing
arrow)` icon to the right of the search bar. Doing so reveals a drop-down menu of detailed filter
and grouping options.

.. note::
   Multiple filter options and grouping options can be selected at once.

> .. image:: reports/reporting-filter-options-dropdown.png
>   :align: center

Reviewer 3: Indent image attributes by one more space, please check all of the images in this doc, there are quite a few that need this adjustment :)
```suggestion
.. image:: reports/reporting-filter-options-dropdown.png
   :align: center
   :alt: The reporting filter drop-down menu of options in the Odoo Subscriptions application.
```

the title field, as well: :guilabel:`Default filter` and :guilabel:`Shared`.

Ticking the check box beside :guilabel:`Default filter` makes the newly-favorited filter the default
option for that reporting page.

Ticking the box beside :guilabel:`Shared` makes the newly-favorited filter available to other users
in the database.

.. note::
   The :guilabel:`Default filter` and :guilabel:`Shared` options are **not** required. Only *one* of
   those options can be selected at a time.

To save that filter(s), click :guilabel:`Save` in the :guilabel:`Favorites` section of the drop-down
filter menu.

When clicked, that saved filter appears beneath the :guilabel:`Favorites` column of the drop-down
filter menu, and a :guilabel:`⭐ (gold star)` icon appears beside the favorite filter's name in the
search bar.

Views
-----

On the :guilabel:`Subscription Analysis`, :guilabel:`MRR Breakdown`, and :guilabel:`MRR Analysis`
reporting pages, three different view options are located in the upper-right corner.

.. note::
   There are *no* other view options available on the :guilabel:`Retention Analysis` reporting page.

The available view options, from left to right, are:

> .. image:: reports/subscriptions-analysis-page-view-options.png
>   :align: center
>   :alt: The different view options available on the Subscriptions Analysis page.
> 
> - :guilabel:`Graph`
> - :guilabel:`Pivot`
> - :guilabel:`List`

Reviewer 3: Suggesting swapping the position of the image with the bullet points, so that there is not this awkward whitespace:
<img width="603" alt="Screenshot 2024-02-07 at 10 18 31 AM" src="https://github.com/odoo/documentation/assets/36018073/073a77f2-d5e0-4543-9560-75acb6f3501a">
```suggestion
- :guilabel:`Graph`
- :guilabel:`Pivot`
- :guilabel:`List`

.. image:: reports/subscriptions-analysis-page-view-options.png
   :align: center
   :alt: The different view options available on the Subscriptions Analysis page.
```

- :guilabel:`Expand all`: all the available rows and columns of the pivot data table
  expand fully.
- :guilabel:`Download xlsx`: the pivot data table is downloaded as an .xlsx file.

List view
~~~~~~~~~

With the list view selected, the subscription metrics being analyzed are displayed in a simple list,
which can be fully customized by using any of the available filters in the drop-down filter menu
(accessible via the :guilabel:`▼ (downward facing arrow)` icon to the right of the search bar).

.. note::
   With list view selected, the :guilabel:`Measures` drop-down menu and :guilabel:`Insert in
   Spreadsheet` button are *not* available.

Measures
--------

Each reporting page has its own metric-specific :guilabel:`Measures` drop-down menu of data-related
options to choose from, located in the upper-left corner, above the visual representation of
metrics.

.. image:: reports/subscriptions-measures-drop-down.png
  :align: center
  :alt: The standard measures drop-down menu in the Odoo Subscriptions application.

When clicked, a series of selectable measures become available, via a drop-down menu. When any of

> the options are selected from a :guilabel:`Measures` drop-down menu, the chosen metrics related
> to that specific measure appear on the reporting page.

Reviewer 3: Early line break
```suggestion
the options are selected from a :guilabel:`Measures` drop-down menu, the chosen metrics related to
that specific measure appear on the reporting page.
```

.. important::
   There are *no* other view options available on the :guilabel:`Retention Analysis` reporting page.

The available view options, from left to right, are:

.. image:: reports/subscriptions-analysis-page-view-options.png
  :align: center
  :alt: The different view options available on the Subscriptions Analysis page.

- :guilabel:`Graph`
- :guilabel:`Pivot`
- :guilabel:`List`

Each view has its own series of related view-specific visual options.

Graph view
~~~~~~~~~~

With the graph view selected the following options appear between the :guilabel:`Search...` bar and
visual representation of the data. These graph-specific options are located to the right of the
:guilabel:`Measures` drop-down menu and :guilabel:`Insert in Spreadsheet button`.

.. image:: reports/subscriptions-graph-specific-options.png
  :align: center
  :alt: The different graph view options in the Odoo Subscriptions application.

The first three options, from left to right, represent different graph-related views. The remaining
options represent different ways to organize and visualize that specific graph-related data.

> From left to right, the specific graph-related view options:

Reviewer 4: ```suggestion
From left to right, the specific graph-related view options are:
```

- :guilabel:`Bar Chart`: showcases the data in a bar chart format.
- :guilabel:`Line Chart`: showcases the data in a line chart format.
- :guilabel:`Pie Chart`: showcases the data in a pie chart format.

Each graph view option has its own series of specific visual options, which are represented by the
available buttons that appear to the right of the selected graph-related view option.

When the :guilabel:`Bar Chart` graph view is selected, the following visual options are available:

- :guilabel:`Stacked`: showcases the data in a stacked visual format.
- :guilabel:`Descending`: showcases the data in descending order.
- :guilabel:`Ascending`: showcases the data in ascending order.

When the :guilabel:`Line Chart` graph view is selected, the following visual options are available:

- :guilabel:`Stacked`: showcases the data in a stacked visual format.
- :guilabel:`Cumulative`: showcases the data in accumulated, increasing format.
- :guilabel:`Descending`: showcases the data in descending order.
- :guilabel:`Ascending`: showcases the data in ascending order.

When the :guilabel:`Pie Chart` graph view is selected, there are no additional visual options.

Pivot view
~~~~~~~~~~

> With the pivot view selected, the subscription metrics being analyzed are displayed in a data table,

Reviewer 4: ```suggestion
With the pivot view selected, the subscription metrics are displayed in a data table,
```

which can be fully customized.

The pivot data table can be customized using the options available in the :guilabel:`Measures`
drop-down menu and/or the filter options available in the filter drop-down menu (accessible via the
:guilabel:`▼ (downward facing arrow)` icon to the right of the :guilabel:`Search...` bar).

There are three pivot-specific options available, located to the right of the :guilabel:`Measures`
drop-down menu and :guilabel:`Insert in Spreadsheet` button.

.. image:: reports/subscriptions-pivot-view-options.png
  :align: center
  :alt: The pivot-specific view options available in the Odoo Subscriptions application.

From left to right, those pivot-specific view options are:

> - :guilabel:`Flip axis`: when clicked, the `x` and `y` axis of the pivot data table flip.
> - :guilabel:`Expand all`: when clicked, all the available rows and columns of the pivot data table
>   expand fully.
> - :guilabel:`Download xlsx`: when clicked the pivot data table is download as an .xlsx file.

Reviewer 4: ```suggestion
- :guilabel:`Flip axis`: the `x` and `y` axis of the pivot data table flip.
- :guilabel:`Expand all`: all the available rows and columns of the pivot data table
  expand fully.
- :guilabel:`Download xlsx`: the pivot data table is downloaded as an .xlsx file.
```

to that specific measure appear on the reporting page.

.. note::
  For more information on the different measures that can be utilized on each reporting page, refer
  to the specific reporting page breakdowns found below in this documentation.

Insert in spreadsheet
---------------------

Beside the :guilabel:`Measures` drop-down menu, there is an :guilabel:`Insert in Spreadsheet`
button.

When clicked, the ability to add the configured data currently being showcased on the reporting page
into a new or pre-existing spreadsheet (or dashboard) becomes available, via a pop-up window.

.. image:: reports/subscriptions-analysis-spreadsheet-popup.png
  :align: center
  :alt: The spreadsheet pop-up window of the Subscriptions Analysis page.

Select the desired option from this pop-up window, then click :guilabel:`Confirm`.

Reporting pages
===============

In the Odoo *Subscriptions* application, there are four different reporting pages available.

To access, analyze, and customize various reports related to subscriptions, navigate to
:menuselection:`Subscriptions app`, and click the :guilabel:`Reporting` drop-down menu in the
header.

> Clicking :guilabel:`Reporting` in the header menu of the *Subscriptions* application, reveals a

Reviewer 4: I don't think this needs to be repeated from the previous paragraph. It's pretty self-explanatory that we're in the Subscriptions app for this doc
```suggestion
Clicking :guilabel:`Reporting` in the header menu reveals a
```

drop-down menu of the following reporting pages:

- :guilabel:`Subscriptions`
- :guilabel:`Retention`
- :guilabel:`MRR Breakdown`
- :guilabel:`MRR Timeline`

Clicking any of those options reveals a separate, fully-customizable reporting page focusing on that
particular aspect of subscription data.

The following is a breakdown of those four specific reporting pages.

Subscriptions analysis

> ======================

Reviewer 4: It would make sense for this section (and all of the following sections) to be H3 headings so that they all fall under the "Reporting pages" section
